### DISCUSSION TITLE
Update at specific window

### DISCUSSION THREAD

**Discussion by thulasidassrinivasan** [fluxcd/flux2#2723] [Q&A] [answered]
Hi Team,

I like the idea of auto image update, It is a wonderful concept.  

I am wondering if there is a way I can set the image update window for e.g., update the images only during 18-19 UTC.  If this is something available, How can i use this functionality. 
https://fluxcd.io/docs/guides/image-update/

**Comment by thulasidassrinivasan** [comment]
@Thaval & @stefanprodan - Any suggestions here please?

**Comment by stefanprodan** [comment, marked as answer]
There is an RFC here that would cover your feature request: https://github.com/fluxcd/flux2/discussions/870#discussioncomment-420896

**Reply by thulasidassrinivasan** [reply]
Great. Is there a date that we can expect a release for this ?

**Comment by kingdonb** [comment]
Another suggestion is to use a `CronJob` resource. You could configure a `CronJob` such that it `resume`s the `ImageUpdateAutomation` at the start of the window, and `suspend` it at the end of the window. I think this workaround is practicable now; this might be a way to get the functionality you're looking for ahead of the RFC that was linked.

We could provide an example in the docs, this is a very common request. I'm not certain how soon we can expect the RFC to be implemented. It is a popular issue, but with a workaround – and it has been up for discussion for quite a while without much progress.
